views/doc.list.html.twig

```diff
@@ -3,36 +3,61 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <article role="article">
+
+        <div id="searchbar" class="clearfix">
+            <input id="docsearch" type="search" placeholder="Type here to search documentation..." role="searchbox"
+                   title="Enter text here and submit to search the documentation"/>
+        </div>
+
+        <h1 role="heading">Book</h1>
```

Comment:
the explicit role should be removed, as that's already the built-in role of a `<h1>`. The explicit `role` attribute should not be used everywhere to be accessible. It should be used in places where HTML itself is not enough.

Reply:
Done.